ext/kv/remote.rs

```diff
@@ -127,6 +129,7 @@ impl<P: RemoteDbHandlerPermissions + 'static> DatabaseHandler
     let options = &self.http_options;
     let client = create_http_client(
       &options.user_agent,
+      options.dns_resolver.clone(),
```

Comment:
On second thought -- maybe this should live on `CreateHttpClientOptions`?

Reply:
I went with a struct instead of a trait (`DnsResolver`), and implemented `Debug`. A trait seemed kind of overkill, idk. Wdyt?